test/table.html

```diff
@@ -0,0 +1,705 @@
+<!DOCTYPE html>
```

Comment:
Since you have so many different test cases, I'd suggest breaking them up into different files. Percy will do a diff for each one, which will make it easier to spot changes.

Also, can you add some RTL tests?